lpa-submissions-e2e-tests/cypress/support/PageObjects/common-page-objects.js

```diff
@@ -0,0 +1,55 @@
+exports.questionTitle = () => {
```

Comment:
I question the value of this file. While it is nice to have things in 1 place, this is only useful if they are reused. It seems the majority of these are used in a single function so perhaps make more sense to have them in those functions instead? Open to arguments otherwise

Reply:
Considered, left in for now.